src/cli/java/org/commcare/util/engine/CommCareConfigEngine.java

```diff
@@ -214,9 +214,11 @@ private void init(String profileRef) {
         }
     }
 
-    public void installAppFromReference(String profileReference) throws UnresolvedResourceException,
+    public void installAppFromReference(String profileReference)
+            throws UnresolvedResourceException,
             UnfullfilledRequirementsException, InstallCancelledException {
-        ResourceManager.installAppResources(platform, profileReference, this.table, true);
+        ResourceManager.installAppResources(platform, profileReference, this.table, true,
+                Resource.RESOURCE_AUTHORITY_LOCAL);
```

Comment:
Yeah, currently the CLI only supports 3 modes:
1) Link to an online CCZ file that gets downloaded and unzipped locally
2) Link to a local CCZ file that gets the same treatment
3) Link to a _local_ profile file 

The CLI _should_ support online _update, but installs I don't think ever rely on a ccpr file from the network.